packages/quick_actions/quick_actions/README.md

```diff
@@ -19,23 +20,40 @@ Initialize the library early in your application's lifecycle by providing a
 callback, which will then be called whenever the user launches the app via a
 quick action.
 
+<?code-excerpt "main.dart (Initialize)"?>
 ```dart
-final QuickActions quickActions = const QuickActions();
-quickActions.initialize((shortcutType) {
-  if (shortcutType == 'action_main') {
-    print('The user tapped on the "Main view" action.');
-  }
-  // More handling code...
+const QuickActions quickActions = QuickActions();
+quickActions.initialize((String shortcutType) {
+  setState(() {
+    shortcut = shortcutType;
```

Comment:
Without the context of the example app, this is significantly less clear than the previous version of the code. I think we'll need to use an excerpt file for this instead of pulling from the example app, since I don't think we can rework the example to be as clear as the previous example.

Reply:
@stuartmorgan  I have simplified this implementation to match closely with the previous version.